test/Base.t.sol

```diff
@@ -27,6 +27,9 @@ abstract contract Base_Test is PRBTest, StdCheats, PRBMathAssertions, PRBMathUti
                                      CONSTANTS
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @dev The minimum value an uint128 number can have.
+    uint128 internal constant MIN_UINT128 = type(uint128).min;
```

Comment:
Unused constant. Perhaps you wanted to define `MIN_UINT256`?

Reply:
Yes, my bad, looks like I mixed up variables when searching where to define it

Reply:
No worries! I fixed this in the commit below.